CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+## How to contribute to CoffeeScript
```

Comment:
I think this file doesn't apply to this repo yet, since we don't have any actual code yet.